packages/flutter/lib/src/material/stepper.dart

```diff
@@ -362,11 +378,19 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
     return false;
   }
 
-  Widget _buildLine(bool visible) {
+  Color _connectorColor(bool isActive) {
+    final ColorScheme colorScheme = Theme.of(context).colorScheme;
+    final Set<MaterialState> states = <MaterialState>{
+      if (isActive) MaterialState.selected else MaterialState.disabled,
+    };
+    return widget.connectorColor?.resolve(states) ?? (isActive ?colorScheme.primary:Colors.grey.shade400);
```

Comment:
This doesn't appear to be backwards compatible. It looks like the color was unconditionally `Colors.grey.shade400`. That would explain the customer testing failures.